Servers/UI/OJS.Servers.Ui.Models/Submissions/Details/SubmissionDetailsResponseModel.cs

```diff
@@ -21,6 +21,8 @@ public class SubmissionDetailsResponseModel : IMapFrom<SubmissionDetailsServiceM
 
         public UserProfileResponseModel User { get; set; } = null!;
 
+        public bool UserIsInRoleForContest { get; set; }
```

Comment:
What does this mean? Also why it is not in the `User` property?

Reply:
This means that the user is either admin or a lecturer in the contest or contest category

Reply:
The service model of the user is used in multiple other endpoints where UserIsInRoleForContest  is not used